protocol/storage/miner.go

```diff
@@ -580,17 +579,12 @@ func (sm *Miner) onCommitFail(ctx context.Context, dealCid cid.Cid, message stri
 		resp.Message = message
 		resp.State = storagedeal.Failed
 	})
-	log.Errorf("commit failure but could not update to deal 'Failed' state: %s", err)
+	sm.log.Errorf("commit failure but could not update to deal 'Failed' state: %s", err)
 }
 
 // currentProvingPeriodPoStChallengeSeed produces a PoSt challenge seed for
 // the miner actor's current proving period.
```

Comment:
Given that you added the `currentProvingPeriodStart` parameter, this comment is no longer true. Rather, it should be rewritten to say:

```
// currentProvingPeriodPoStChallengeSeed produces a PoSt challenge 
// using randomness from the chain at the provided block height
```